Heroku/README.md

```diff
@@ -0,0 +1,23 @@
+# Heroku
+
+This is an app for the Heroku. It contains a dashboard for an overview, performance, and routers. 
+
+Heroku logs are collected through [log drains](https://devcenter.heroku.com/articles/log-drains). Ideally, these logs should be forwarded to a [Sumo Logic HTTPS Source](https://help.sumologic.com/Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source). 
+
+Additional documentation from Heroku on required logging:
+
+* [HTTP Routing - Heroku Router Log Format](https://devcenter.heroku.com/articles/http-routing#heroku-router-log-format)
+* [Heroku Labs: log-runtime-metrics](https://devcenter.heroku.com/articles/log-runtime-metrics)
+
+
+## Setup
+
+### Update Source Categories
+
+Update source categories to your appropriate one(s):
+
+_sourceCategory=\*heroku\*
```

Comment:
It would really be helpful in getting these "as real apps" to change the part that must be replaced to something in the form of
$$REPLACE_FIELD
Emphasis on the "$$" as this is required in an app to be importable via the interface AND will help facilitate the transition in the future. 
I would recommend that it be changed to "$$herokulogs" with the instruction that it is required to be changed.